src/languages/sql.js

```diff
@@ -655,11 +661,11 @@ export default function(hljs) {
     },
     contains: [
       {
-        begin: regex.either(...COMBOS),
+        begin: regex.either(...generateComboKeywords(COMBOS)),
```

Comment:
This can't use the split version or it'll give the same false positives on `key`.... only the "internal" `keywords` needs the full expanded list (for matching individual words inside larger keyword combos)... but I have a feeling then we're going to find that compound keywords spread across multiple lines don't work, ugh.

Reply:
i believe ive managed to do it 
ive even added support for newline but i had to make a new comboslist @joshgoebel  please review 
now 
![image](https://github.com/user-attachments/assets/e53cc2d1-97b1-40ec-a08f-0fef60d2394d)